packages/block-editor/src/components/gradient-picker/control.js

```diff
@@ -23,14 +24,12 @@ export default function GradientPickerControl( {
 	label = __( 'Gradient Presets' ),
 	...props
 } ) {
-	const { gradients = [], disableCustomGradients } = useSelect(
-		( select ) =>
-			pick( select( 'core/block-editor' ).getSettings(), [
-				'gradients',
-				'disableCustomGradients',
-			] ),
-		[]
-	);
+	const gradients =
+		useSelect(
```

Comment:
Does it make sense to have a `usePreset('gradients')` hook?

Reply:
:) https://github.com/WordPress/gutenberg/pull/24761#discussion_r475537656

Reply:
I see! I'm only 10 days behind :runner: :dash: